src/Components/DayForm.js

```diff
@@ -0,0 +1,146 @@
+import Button from './Button';
+import ScreenReaderOnly from './Styles/ScreenReaderOnly';
+import SpaceBetween from './Styles/SpaceBetween';
+import Center from './Styles/Center';
+import styled from 'styled-components';
+import { IoIosAddCircleOutline } from 'react-icons/io';
+
+export default function DayForm({ onSubmit, day }) {
+  function handleSubmit(event) {
+    event.preventDefault();
+    const form = event.target;
+    const { exercise, weight, repetitions, sets } = form.elements;
+    const newDay = {
+      day,
+      exercise: exercise.value,
+      weight: weight.value,
+      repetitions: repetitions.value,
+      sets: sets.value,
+    };
+    onSubmit(newDay);
+  }
+
+  return (
+    <>
+      <FormBox>
+        <ScreenReaderOnly id="formSpan" Create your workout sessions />
```

Comment:
I don't think this actually works as `Create your workout sessions` are now multiple props and not a string or so. Please check how to pass or use this :)